### ISSUE TITLE
RockMongo Display Issues

### ISSUE THREAD

**Issue by sinkingtheyap** [iwind/rockmongo#120]
RockMongo v1.1.6 seems to have issues when trying to display large documents, espically if those documents contain large arrays of data. Is anyone else getting this error. I have tested this with MongoChef and the data is there and valid and usable in my code but just can't be displayed in RockMongo.

**Comment by martonic** [comment]
Yes, it's bizarre.  The display simply chokes.  If I add a few small documents to the collection then it displays everything.  Otherwise, the process quits after a few seconds and displays nothing.  This is with version 1.1.7.  As far as I can tell, this can happen with any collection having a document that requires more than around one-hundred lines of text to display.  It does not take much to trigger this issue.  I have not tried to pin it down further or debug it myself.

**Comment by stefanocudini** [comment]
Vote on this issue please #133
